src/ClimaLSM.jl

```diff
@@ -204,9 +204,12 @@ per component model.
 function make_set_initial_cache(land::AbstractLandModel)
     components = land_components(land)
     # These functions also call update_aux
+    (atmos, radiation) = get_drivers(land)
+    update_drivers! = make_update_drivers(atmos, radiation)
     set_initial_cache_function_list =
         map(x -> make_set_initial_cache(getproperty(land, x)), components)
     function set_initial_cache!(p, Y0, t0)
+        update_drivers!(p, t0)
```

Comment:
I dont necessarily like this, but not sure how to avoid. Basically, each component model's `set_initial_cache!` calls `update_drivers!`, which isnt a huge problem (just duplicative, but only done once), but in the soil canopy model has a weird issue. TBD but wip.

Reply:
I opened this PR to address this - let me know if you think that the PR describes the right thing. I think if we can get rid of `CanopyRadiativeFluxes` as a driver (and instead use `PrescribedRadiativeFluxes`), we wont have this issue, so the problem really is tied to why we came up with that in the first place and solving that more elegantly?
https://github.com/CliMA/ClimaLSM.jl/pull/446

Reply:
That PR description looks good to me!